src/pages/index.js

```diff
@@ -335,11 +362,105 @@ const IndexPage = (props) => {
+                                justifyContent: 'center',
+                                alignItems: 'center',
+                            }}
+                        >
+                            <img
+                                src={thankYou[6]?.replace(/['"]+/g, '')}
+                                alt='Random contributor image'
+                                width={isDesktop ? 100 : isMobile ? 50 : 90}
+                                height={isDesktop ? 100 : isMobile ? 50 : 90}
+                                style={{
+                                    marginTop: 'auto',
+                                    marginBottom: 'auto',
+                                }}
+                            />
+                        </Box>
+                        <Box
+                            sx={{
+                                fontSize: isMobile ? 'small' : 'medium',
+                            }}
+                        >
+                            Thank you{' '}
+                            {thankYou.filter((item) => item?.trim() === '')
+                                .length === 0 && (
+                                <a
+                                    target='_blank'
+                                    href={thankYou[5]?.replace(/['"]+/g, '')}
+                                >
+                                    {thankYou[3] === true
+                                        ? thankYou[3]?.replace(/['"]+/g, '')
+                                        : thankYou[2]?.replace(/['"]+/g, '')}
+                                </a>
+                            )}
+                            <br />
+                            for making {thankYou[7]?.replace(/['"]+/g, '')} pull
+                            requests(s)
+                            <br />
+                            to the{' '}
+                            {thankYou[8]
+                                ?.replace(/['"]+/g, '')
+                                .split(/\s+/)
+                                .filter(Boolean)
+                                .map((repo, idx) => (
+                                    <>
+                                        <a
+                                            target='_blank'
+                                            href={`https://github.com/${repo}`}
+                                        >
+                                            {repo.split('/')[1]}
+                                        </a>
+                                        {idx <
+                                        thankYou[8].split(' ').length - 1 ? (
+                                            <>
+                                                ,
+                                                <br />
+                                            </>
+                                        ) : (
+                                            ' '
+                                        )}
```

Comment:
There is a comma and a new line return at the end of the pull request list, is it expected?

https://deploy-preview-141--contributor-spotlight.netlify.app/

<img width="425" alt="image" src="https://github.com/jenkins-infra/contributor-spotlight/assets/91831478/101c4c80-08ec-4457-8afc-1221c2121d7b">

Reply:
I have removed this extra comma